nessie/jobs/create_oua_schema.py

```diff
@@ -64,17 +52,31 @@ def migrate_oua_sftp_data(self):
                     destination_key = source_key.replace(oua_slate_sftp_path, oua_daily_dest_path)
                     if not s3.copy(s3_protected_bucket, source_key, s3_protected_bucket, destination_key):
                         raise BackgroundJobError(f'Copy from SFTP location {source_key} to daily OUA destination {destination_key} failed.')
+            external_schema = app.config['REDSHIFT_SCHEMA_OUA']
+            redshift.drop_external_schema(external_schema)
+            resolved_ddl = resolve_sql_template('create_oua_schema_template.sql')
+            if redshift.execute_ddl_script(resolved_ddl):
+                verify_external_schema(external_schema, resolved_ddl)
+                self.create_rds_tables_and_indexes()
+                app.logger.info('OUA Slate RDS indexes created.')
+                return 'OUA schema creation job completed.'
+
+            else:
+                raise BackgroundJobError('OUA Slate schema creation job failed.')
+
         else:
-            raise BackgroundJobError('No OUA files found in SFTP location today. Skipping OUA data refresh')
+            app.logger.warn('No OUA files found in SFTP location today. Skipping OUA data refresh')
```

Comment:
@johncrossman Looks good, but I might have it `return` this message rather than `app.logger.warn`. If we return a string value, then the message should show up on the Nessie dashboard next to the green status, which is handy for checking whether any files were found.

Reply:
Thanks @paulkerschen  I'll amend.